bindings/rust/s2n-tls/src/raw/error.rs

```diff
@@ -70,6 +70,27 @@ impl Fallible for isize {
     }
 }
 
+impl Fallible for u64 {
+    type Output = Self;
+
+    /// Converts a u64 to a Result by checking for the maximum value.
+    ///
+    /// If a method that returns an unsigned int is fallible,
+    /// then the -1 error result wraps around to the maximum value.
+    /// The maximum value must not be possible otherwise.
+    ///
+    /// For example, [`s2n_connection_get_delay`] can't return
```

Comment:
So I guess what you're trying to say is that s2n_connection_get_delay isn't Fallible? I am not following this explanation.

Reply:
s2n_connection_get_delay is Fallible, just in a really unfortunate way. I updated the comment-- does that makes it clearer?